plugins/scaffolder-backend/src/scaffolder/actions/builtin/debug/log.examples.ts

```diff
@@ -48,4 +48,20 @@ export const examples: TemplateExample[] = [
       ],
     }),
   },
+  {
+    description: 'List the workspace directory with file contents',
+    example: yaml.stringify({
+      steps: [
+        {
+          action: 'debug:log',
+          id: 'write-workspace-directory',
+          name: 'List the workspace directory with file contents',
+          input: {
+            listWorkspace: true,
```

Comment:
Potential option here since there's currently a hidden dependency on `listWorkspace` from `listFileContents`: Might make sense to have this be `listWorkspace?: 'with-contents' | boolean` or something like that instead